pages.fr/common/reboot.md

```diff
@@ -0,0 +1,20 @@
+# reboot
```

Comment:
You should move these pages to the `linux/` folder.

Reply:
` pages.fr/common/reboot.md -> pages.fr/linux/reboot.md` in (d381a40ba8981bccdcf9fad356be6b8b85de9616)

Reply:
@turnipsoup `shutdown` too.

Reply:
That file already exists and it is pretty much the same, so I removed my version from my PR. (45d1a3696b9db143b950fef5ec90cae01b84705d)